Assets/MixedRealityToolkit.Tests/PlayModeTests/InteractableTests.cs

```diff
@@ -439,6 +439,61 @@ public IEnumerator TestDestroy()
             Assert.AreEqual(propBlock.GetColor("_Color"), FocusColor);
         }
 
+        [UnityTest]
+        /// <summary>
+        /// Tests button depth and focus state after enabling, disabling and re-enabling Interactable 
+        /// internally via SetDisable(true). The focus state after re-enabling should be false and button
+        /// depth should be in its default position.  This test is specifically addressing behavior described 
+        /// in issue 4967.
+        /// </summary>
+        public IEnumerator TestFocusStateOnDisableEnableInteractable()
+        {
+            var inputSimulationService = PlayModeTestUtilities.GetInputSimulationService();
+            int numSteps = 20;
+            Vector3 p1 = new Vector3(0.0f, 0f, 0f);
+            Vector3 p2 = new Vector3(0.05f, 0f, 0.51f);
+            Vector3 p3 = new Vector3(0.0f, 0f, 0.0f);
+
+            // Load the Model_PushButton interactable prefab
+            Interactable interactable;
+            Transform innerCylinderTransform;
+
+            InstantiatePressButtonPrefab(
+                new Vector3(0.0f, 0.0f, 0.5f),
+                new Vector3(-90f, 0f, 0f),
+                out interactable,
+                out innerCylinderTransform);
+
+            // OnClick, disable Interactable which sets Enabled property to false
+            interactable.OnClick.AddListener(() => { interactable.SetDisabled(true); });
+
+            // Get start position of the inner cylinder before button is pressed
+            Vector3 innerCylinderStartPosition = innerCylinderTransform.localPosition;
+
+            // Move the hand forward to press button
+            yield return PlayModeTestUtilities.ShowHand(Handedness.Right, inputSimulationService);
```

Comment:
Instead of using playmodetestutilities, it would be nicer to just use TestHand class, it will make your code more concise, and TestHand hides things like waiting for frames.

Reply:
Updated the test to use TestHand.